frontend/src/components/associations/Home.tsx

```diff
@@ -2,16 +2,42 @@ import React from "react";
 import { PageTitle } from "../utils/PageTitle";
 import Container from "react-bootstrap/Container";
 import { Loading } from "../utils/Loading";
+import { api, useBetterPaginatedQuery } from "../../services/apiService";
+import { ErrorMessage } from "../utils/ErrorPage";
+import { AssociationPageRenderer } from "./page/Show";
 
 export const AssociationHome = ({ association }) => {
-    if (!association) {
-        return <Loading />;
-    }
+    const { resolvedData: data, error, status } = useBetterPaginatedQuery<any>(
+        [
+            "association.page.list.homepage",
+            association.id,
+            { title: "Accueil" },
+        ],
+        api.pages.list
+    );
 
-    return (
-        <Container>
-            <PageTitle>{association.name}</PageTitle>
-            <p>Home page !!!</p>
+    return status === "loading" ? (
+        <Loading />
+    ) : status === "error" ? (
+        <ErrorMessage>{error}</ErrorMessage>
+    ) : data.count === 0 ? (
+        <Container className={"text-center"}>
+            <p style={{ fontSize: "10em" }} className={"m-0"}>
+                🏡
```

Comment:
This triggers a warning:  Emojis should be wrapped in `span`, have role="img", and have an accessible description with aria-label or aria-labelledby